libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -67,6 +67,12 @@ const AP_Param::GroupInfo AP_Vehicle::var_info[] = {
     AP_SUBGROUPINFO(efi, "EFI", 9, AP_Vehicle, AP_EFI),
 #endif
 
+#if !HAL_MINIMIZE_FEATURES && APM_BUILD_TYPE(APM_BUILD_Rover) // this should really be HAL_AIS_ENABLED
+    // @Group: AIS_
+    // @Path: ../AP_AIS/AP_AIS.cpp
+    AP_SUBGROUPINFO(ais, "AIS_",  10, AP_Vehicle, AP_AIS),
+#endif
```

Comment:
@peterbarker @hendjoshsr71 I have reworked this to the dummy methods approach as in airspeed, any thoughts about how to get round the param table? Everything else works, not sure the flash cost yet......

Reply:
We could change to a pointer table of course and load at init, but that seems overly complex.